### DISCUSSION TITLE
Lab Assignment #2 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#10] [Assignments]
## Due 4/20

Please use this thread to post your [lab assignment's #2](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week2/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/23S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### Note 
If your map is not on the main `index.html` page, be sure to submit that sub-page's link instead:

#### For example:
https://albertkun.github.io/23s-asiaam-191a-sample/week2/map.html

If you only have one page, then you can just submit the `index.html` link as per usual.

**Comment by czhangy** [comment]
https://czhangy.github.io/ASIA-AM-191A/week2/index.html

**Comment by catherinehhu** [comment]
https://catherinehhu.github.io/191a-labs/week2/map.html

**Comment by SageLuong** [comment]
https://sageluong.github.io/23s-asianam-191a-labs/week2/restaurants.html

**Comment by zacharypakin** [comment]
https://zacharypakin.github.io/aa191a-23s/week02/index.html

**Comment by danigold1020** [comment]
https://danigold1020.github.io/23s-asianam-191a-labs/week02/index.html

**Comment by vickydelk** [comment]
https://vickydelk.github.io/aa191-assignments/week2/index.html

**Comment by paige-lee** [comment]
https://paige-lee.github.io/aa191a-labs/week2/index.html

**Comment by qatle** [comment]
https://trinhqale.github.io/AM191A-labs/week2/index.html

**Comment by maizaha** [comment]
https://maizaha.github.io/23s-asiaam-191a-labs/week2/index.html

**Comment by emai14** [comment]
https://emai14.github.io/23S-ASAM191A-Labs/week02/

**Comment by chrysanthemma** [comment]
https://chrysanthemma.github.io/asainam191a-labs/week1/index.html

**Reply by joanneluong** [reply]
https://joanneluong.github.io/aas191a-labs/week1/index.html

**Comment by kmtsujimoto** [comment]
https://kmtsujimoto.github.io/23s-asiaam-191a-labs/week02/index.html

**Comment by shintuzd** [comment]
https://shintuzd.github.io/23s-asiaam-191a-assignments/week2/maps.html

**Comment by ariscorpus** [comment]
https://ariscorpus.github.io/23s-asianam-191a-labs/week2/

**Comment by angelicaalacantar** [comment]
https://angelicaalacantar.github.io/aas191a-23s/week2/index.html

**Comment by cassiuscarandang** [comment]
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week2/index.html

**Comment by jamesfalkenroth** [comment]
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week2/index.html

**Comment by mandymwlai** [comment]
https://mandymwlai.github.io/AA-191-lab/week1/

**Comment by alanamalika** [comment]
https://alanamalika.github.io/23s-asianam-191a-labs/week02/index.html

**Comment by amandatlxu** [comment]
https://amandatlxu.github.io/

**Comment by rebeccaksuen** [comment]
https://github.com/rebeccaksuen/23s-asiaam-191a-labs/tree/main/Week%202

**Comment by DylanJWan** [comment]
https://dylanjwan.github.io/asiaam-191a-assignments/portfolio/maps

**Comment by enyi-emuka** [comment]
https://enyi-emuka.github.io/asiaam191a_labs/week2/index.html

**Comment by JonathanGLopez** [comment]
https://jonathanglopez.github.io/Asiaam191aLB1/Week2/index.html

**Comment by cassidytu** [comment]
https://cassidytu.github.io/asam191a-labs/week2/map.html

**Comment by trufflelover** [comment]
https://trufflelover.github.io/23s-asiaam-191a-labs/week2/
